packages/layers/src/xr-3d-layer/xr-3d-layer.js

```diff
@@ -285,10 +289,15 @@ const XR3DLayer = class extends Layer {
       // Need to flatten for shaders.
       const normals = paddedClippingPlanes.flatMap(plane => plane.normal);
       const distances = paddedClippingPlanes.map(plane => plane.distance);
-      model
-        .setUniforms({
+      //HACK: null textures will throw errors, so we just set them all to the first texture FOR THE VERY SHORT TERM!
+      for (const key in textures) {
+        if (!textures.volume0) throw new Error('Bad texture state!');
+        if (!textures[key]) textures[key] = textures.volume0;
+      }
```

Comment:
Ok so I see that this is definitely a problem, and without this things do not work.  I wonder about two things

1. We probably shouldn't update this in place since `textures` is in `state`
2. Why doesn't this affect the rendering?  It seems like making the pass on that texture should render something?